templates/v2/pro/list/_update_right_button.html.twig

```diff
@@ -0,0 +1,36 @@
+{% set currentUserRelay = relay ? user.userRelay(relay) : null %}
+{% if not currentUserRelay %}
+    {% set currentUserRelay = user.userRelays|length == 0 ? user.userRelays[0] : null %}
+{% endif %}
+
+{% set buttons = [
+    {
+        icon: 'user',
+        right: constant('App\\Entity\\MembreCentre::TYPEDROIT_GESTION_BENEFICIAIRES'),
+    },
+    {
+        icon: 'user-cog',
+        right: constant('App\\Entity\\MembreCentre::TYPEDROIT_GESTION_MEMBRES'),
+    },
+] %}
+
+{% for button in buttons %}
+    <button class="btn border-0"
+        {% if relay %}
+            {{ stimulus_controller('ajax-toggle', {
+                url: path('pro_toggle_right', {id: user.subjectMembre.id, relayId: relay.id, right: button.right}),
+            }) }}
+            {{ stimulus_action('ajax-toggle', 'toggle') }}
```

Comment:
With a turbo frame, you dont even need any controller

Reply:
I'm not sure, why don't u like this controller ?